content/pt-br/docs/concepts/configuration/secret.md

```diff
@@ -0,0 +1,2141 @@
+como senhas, tokens ou chaves. Este tipo de informação poderia, em outras
+circunstâncias, ser colocada diretamente em uma configuração de
+{{< glossary_tooltip term_id="pod" >}} ou em uma
+{{< glossary_tooltip text="imagem de contêiner" term_id="image" >}}. O uso de
+Secrets evita que você tenha de incluir dados confidenciais no seu código.
+
+<!--
+Because Secrets can be created independently of the Pods that use them, there
+is less risk of the Secret (and its data) being exposed during the workflow of
+creating, viewing, and editing Pods. Kubernetes, and applications that run in
+your cluster, can also take additional precautions with Secrets, such as
+avoiding writing confidential data to nonvolatile storage.
+-->
+
+Secrets podem ser criados de forma independente dos Pods que os consomem. Isto
+reduz o risco de que o Secret e seus dados sejam expostos durante o processo de
+criação, visualização e edição ou atualização de Pods. O Kubernetes e as
+aplicações que rodam no seu cluster podem também tomar outras precauções com
+Secrets, como por exemplo evitar a escrita de dados confidenciais em local de
+armazenamento persistente (não-volátil).
+
+<!-- 
+Secrets are similar to {{< glossary_tooltip text="ConfigMaps" term_id="configmap" >}}
+but are specifically intended to hold confidential data.
+-->
+
+Secrets são semelhantes a
+{{< glossary_tooltip text="ConfigMaps" term_id="configmap" >}}, mas foram
+especificamente projetados para conter dados confidenciais.
+
+<!--
+{{< caution >}}
+Kubernetes Secrets are, by default, stored unencrypted in the API server's underlying data store (etcd). Anyone with API access can retrieve or modify a Secret, and so can anyone with access to etcd.
+Additionally, anyone who is authorized to create a Pod in a namespace can use that access to read any Secret in that namespace; this includes indirect access such as the ability to create a Deployment.
+
+In order to safely use Secrets, take at least the following steps:
+
+1. [Enable Encryption at Rest](/docs/tasks/administer-cluster/encrypt-data/) for Secrets.
+2. Enable or configure [RBAC rules](/docs/reference/access-authn-authz/authorization/) that
+   restrict reading data in Secrets (including via indirect means).
+3. Where appropriate, also use mechanisms such as RBAC to limit which principals are allowed to create new Secrets or replace existing ones.
+
+{{< /caution >}}
+-->
+
+{{< caution >}}
+Os Secrets do Kubernetes são, por padrão, gravados não-encriptados no sistema
+de armazenamento de dados utilizado pelo servidor da API (etcd). Qualquer pessoa
+com acesso à API ou ao etcd consegue obter ou modificar um Secret.
+Além disso, qualquer pessoa que possui autorização para criar Pods em um namespace
+consegue utilizar este privilégio para ler qualquer Secret naquele namespace. Isso
+inclui acesso indireto, como por exemplo a permissão para criar Deployments.
+
+Para utilizar Secrets de forma segura, siga pelo menos as instruções abaixo:
```

Comment:
This enumerated list is repeated, needs to be 1, 2, 3...

Reply:
About this, the enumerated lists are rendered correctly with the use of `1.` repeated (due to markdown). I tend to use this style to avoid re-numbering all items when a new item is added. Do you think it would be better to have the numbers in code reflect the rendered numbers?